go.mod

```diff
@@ -79,4 +79,7 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace golang.org/x/net => golang.org/x/net v0.24.0
+replace (
+	github.com/open-feature/open-feature-operator/apis => github.com/bacherfl/open-feature-operator/apis v0.0.0-20240508085321-08cc258ed722
```

Comment:
Since we merged https://github.com/open-feature/open-feature-operator/pull/635 , you should be able to test things easily. Anyway please check the replacement :)

Reply:
indeed, now that #635 is merged this replace is not needed anymore :)